Projects/Playground/Playground.Wpf/Playground.Wpf.csproj

```diff
@@ -160,6 +160,10 @@
       <Project>{49f965f5-5824-4bbe-94ea-0178f848abdb}</Project>
       <Name>MvvmCross</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\MvvmCross\MvvmCross.Platforms.Wpf.csproj">
```

Comment:
Should these fixes for WPF sample be pulled out to a separate PR so they can be applied to 6.0.1?

Reply:
It is already there: https://github.com/MvvmCross/MvvmCross/blob/develop/Projects/Playground/Playground.Wpf/Playground.Wpf.csproj#L163

Reply:
yep, so I guess this comes down to how we need to manage PRs/Commits across multiple banches. Do we need to rebase the 6.1 branch to pick up the commit to develop, so that the changes don't appear in this PR?

Reply:
Either that or just get it in through this PR.

Reply:
True, although I guess my point is really to try to define what the process is, so that it is clear to everyone.

Reply:
I did the rebase on 6.1 and this branch, so i'm not sure how this ended up here.

Reply:
Do we need to rebase 6.1 onto develop again?

Reply:
Done